<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>简易数制转换计算器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cvtpushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;

</string>
    </property>
    <property name="text">
     <string>转换</string>
    </property>
   </widget>
   <widget class="QLabel" name="remindlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#E5F1FB;
font: 75 14pt &quot;微软雅黑&quot;;
color:rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string>欢迎使用简易数制转换计算器！</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>转换数字</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox1">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="currentText">
        <string>请选择</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>请选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>26进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>27进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>31进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>34进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36进制</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="inputlineEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;在此处输入待转换数字&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>128</number>
       </property>
       <property name="placeholderText">
        <string>在此输入待转换数字</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <widget class="QComboBox" name="comboBox2">
       <property name="styleSheet">
        <string notr="true">font: 25 14pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <item>
        <property name="text">
         <string>请选择</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>26进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>27进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>28进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>29进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>31进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>33进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>34进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>35进制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36进制</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="outputlineEdit_1">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="outputlineEdit_2">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑 Light'; font-size:12pt; font-weight:24; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;转换结果1:数组&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;</string>
       </property>
       <property name="text">
        <string>转换结果2:栈</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 18pt &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>  常用进制转换结果</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:rgb(170, 170, 255)</string>
           </property>
           <property name="text">
            <string>8进制</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="to2lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:rgb(170, 170, 255)</string>
           </property>
           <property name="text">
            <string>2进制</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:rgb(170, 170, 255)</string>
           </property>
           <property name="text">
            <string>16进制</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:rgb(170, 170, 255)</string>
           </property>
           <property name="text">
            <string>10进制</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="to10lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="to8lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="to16lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 25 12pt &quot;微软雅黑 Light&quot;;
background-color:white;</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_CE">
         <property name="font">
          <font>
           <family>微软雅黑 Light</family>
           <pointsize>18</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;

</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QPushButton" name="pushButton_back">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>←</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_C">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_D">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_E">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_F">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_8">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_9">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_A">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;

</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_B">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_6">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_7">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_0">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="pushButton_1">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">font: 25 18pt &quot;微软雅黑 Light&quot;;
</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>开始(&amp;S)</string>
    </property>
    <addaction name="helpaction"/>
    <addaction name="separator"/>
    <addaction name="authoraction"/>
    <addaction name="declareaction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="action_C"/>
    <addaction name="action_C_2"/>
    <addaction name="action_V"/>
   </widget>
   <addaction name="menu_S"/>
   <addaction name="menu_E"/>
  </widget>
  <action name="action_H">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>作者(&amp;A)</string>
   </property>
  </action>
  <action name="helpaction">
   <property name="text">
    <string>使用|查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>查看帮助(&amp;H)</string>
   </property>
  </action>
  <action name="authoraction">
   <property name="text">
    <string>作者(&amp;A)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>声明(&amp;D)</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
  </action>
  <action name="action_C_2">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
  </action>
  <action name="action_V">
   <property name="text">
    <string>粘贴(&amp;V)</string>
   </property>
  </action>
  <action name="declareaction">
   <property name="text">
    <string>声明(&amp;D)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
